parm/soca/obs/config/insitu_profile_bathy.yaml

```diff
@@ -0,0 +1,37 @@
+obs space:
+  name: insitu_profile_bathy
+  obsdatain:
+    engine:
+      type: H5File
+      obsfile: !ENV ${DATA}/obs/${OPREFIX}insitu_profile_bathy.${PDY}${cyc}.nc4
+  obsdataout:
+    engine:
+      type: H5File
+      obsfile: !ENV ${DATA}/diags/insitu_profile_bathy.${PDY}${cyc}.nc4
+  simulated variables: [waterTemperature, salinity]
+  observed variables: [waterTemperature, salinity]
+  io pool:
+    max pool size: 1
+obs operator:
+  name: Composite
+  components:
+  - name: InsituTemperature
+    variables:
+    - name: waterTemperature
+#  - name: VertInterp
+#    observation alias file: ./obsop_name_map.yaml
+#    - name: salinity
+#    vertical coordinate: sea_water_depth
+#    observation vertical coordinate: depth
+#    interpolation method: linear
+obs error:
+  covariance model: diagonal
+obs filters:
+  # Passivate obs where ocean fraction is > 90%
```

Comment:
I thought the plan was to "passivate" all insitu obs? That's a weird logic for a filter, I do realize this might have been copy pasted from stuff I did eons ago, but that doesn't make it right.